daml-lf/spec/daml-lf-1.rst

```diff
@@ -724,6 +724,9 @@ built-in function ``F`` comes with a fixed type, which we will denote
 as ``𝕋(F)``. See the `Built-in functions`_ section for the complete
 list of built-in functions and their respective types.
 
+Equality on types is defined as structural equality and is only defined on
```

Comment:
This

Reply:
Fair point, I’ve removed it for now. I think there is still something different about the equality used in the type system and the equality used in the runtime check in `from_any` but I don’t recall how other systems define this properly and I agree that the current wording is more confusing than helpful.